src/app/board/board.component.html

```diff
@@ -1,8 +1,8 @@
 <app-header></app-header>
-<div class="container-fluid container-cards-pf">
+<div id="board_topWorkItems" class="container-fluid container-cards-pf">
   <h3>Top WorkItems</h3>
   <div class="grid grid-pad">
-    <div *ngFor="let workItem of workItems" (click)="gotoDetail(workItem)" class="col-1-4">
+    <div *ngFor="let workItem of workItems" (click)="gotoDetail(workItem)" class="col-1-4" id="board_topWorkItemList">
```

Comment:
@SMahil if you run the app and add a few items, then look at the source you will see that @sanbornsen is correct. This will create duplicate IDs. 

For now you will have to remove any IDs in an ngFor.